BondarenkoBE/Why not Leak Cheaker or Valgrind.txt

```diff
@@ -0,0 +1,3 @@
+Because:
```

Comment:
Compiles perfectly
![image](https://user-images.githubusercontent.com/19354025/54873453-a1de5f00-4de7-11e9-830f-f1bfc3091a6d.png)

Reply:
"If the program is compiled, it does not mean that it works correctly" - one great programmer (c)
For some reason, the library string contains memcpy and memmove in VS on Windows, but does not contain them in Leak Cheaker (Linux). The solution was to use string.h instead of string.